repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/minihub-main.ftl

```diff
@@ -3,89 +3,95 @@
 <@hst.defineObjects />
 <@hst.setBundle basename="uk.nhs.hee.web.global"/>
 
-<#-- @ftlvariable name="hstRequestContext" type="org.hippoecm.hst.core.request.HstRequestContext" -->
 <#-- @ftlvariable name="document" type="uk.nhs.hee.web.beans.MiniHub" -->
-<#-- @ftlvariable name="minihubName" type="java.lang.String" -->
-<#-- @ftlvariable name="currentGuidance" type="uk.nhs.hee.web.beans.Guidance" -->
-<#-- @ftlvariable name="previousGuidance" type="uk.nhs.hee.web.beans.Guidance" -->
-<#-- @ftlvariable name="nextGuidance" type="uk.nhs.hee.web.beans.Guidance" -->
-<#-- @ftlvariable name="accessFromRootHub" type="java.lang.Boolean" -->
+<#-- @ftlvariable name="currentSection" type="uk.nhs.hee.web.beans.Guidance" -->
+<#-- @ftlvariable name="previousSection" type="uk.nhs.hee.web.beans.Guidance" -->
+<#-- @ftlvariable name="nextSection" type="uk.nhs.hee.web.beans.Guidance" -->
+<#-- @ftlvariable name="miniHubSections" type="java.util.List" -->
 
-<#assign accessWithEndSlash=hstRequestContext.servletRequest.requestURI?endsWith("/")/>
 <#if document??>
     <div class="nhsuk-width-container">
         <main id="maincontent" role="main" class="nhsuk-main-wrapper">
             <div class="nhsuk-grid-row">
                 <div class="nhsuk-grid-column-two-thirds">
                     <h1>
-                      <span role="text">${currentGuidance.title}
-                        <span class="nhsuk-caption-xl nhsuk-caption--bottom">
-                          ${document.title}
+                        <span role="text">${currentSection.document.title}
```

Comment:
<#if currentSection??> is missing

Reply:
No longer relevant as check is added in commit [2787488](https://github.com/Manifesto-Digital/hee-cms-platform/pull/180/commits/27874883c5218a6af369a532d6f64db0a1a93abd) ?